src/OpenTelemetry.Instrumentation.Grpc/OpenTelemetry.Instrumentation.Grpc.csproj

```diff
@@ -0,0 +1,16 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.1</TargetFrameworks>
```

Comment:
netstandard2.0? why 2.1

Reply:
alsmo missing net452?

Reply:
grpc only works for netcoreapp3.1: https://github.com/grpc/grpc-dotnet/blob/master/src/Grpc.Net.Client/Grpc.Net.Client.csproj#L9